packages/harmony/src/foundations/corner-radius/CornerRadius.stories.mdx

```diff
@@ -0,0 +1,56 @@
+import { Paper, Flex, Text } from 'components'
+import { Meta, Title, Unstyled } from '@storybook/blocks'
+import { useTheme } from '@emotion/react'
+
+<Meta title='Foundations/CornerRadius' />
+
+<Title>Corner Radius</Title>
+
+- [Overview](#overview)
+- [Usage and examples](#usage-and-examples)
+
+## Overview
+
+Corner radius variables are used to give sharp edges a more subtle, rounded effect and impact the overall look and feel of Harmony components.
+
+export function CornerRadiusExample(props) {
+  const { cornerRadius } = props
+  const theme = useTheme()
+  const cornerRadiusValue = theme.cornerRadius[cornerRadius]
+  return (
+    <Flex
+      alignItems='center'
+      justifyContent='center'
+      direction='column'
+      gap='s'
+      h={108}
+      w={108}
+      css={{
+        borderRadius: cornerRadiusValue,
+        border: '1px solid #000'
+      }}
+    >
+      <Text>{cornerRadius}</Text>
+      <Text>{cornerRadiusValue}px</Text>
+    </Flex>
+  )
+}
+
+<Unstyled>
+  <Paper p='xl' justifyContent='space-between'>
+    {['xs', 's', 'm', 'l', 'xl', '2xl'].map((cornerRadius) => (
+      <CornerRadiusExample key={cornerRadius} cornerRadius={cornerRadius} />
```

Comment:
Where is this `CornerRadiusExample` coming from?

Reply:
Should be defined in the mdx doc right above. So cool that is possible

Reply:
ah im blind. yes it is lol